src/java/com/puppetlabs/http/client/impl/JavaClient.java

```diff
@@ -57,8 +58,7 @@
 import java.security.NoSuchAlgorithmException;
 import java.security.cert.CertificateException;
 import java.security.cert.X509Certificate;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
ARGH Intellij-- yes, agreed. Is there a setting for that (to tell Intellij not to do that when automatically importing things)?